.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,42 +1,18 @@
-<!--
-Thank you for contributing!
-Before you submit this PR we'd like to make sure you are aware of our technical requirements and best practices:
-
-* https://github.com/jenkinsci/helm-charts/blob/main/CONTRIBUTING.md#technical-requirements
-* https://helm.sh/docs/chart_best_practices/
-
-For a quick overview across what we will look at reviewing your PR, please read our review guidelines:
-
-* https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md
-
-Following our best practices right from the start will accelerate the review process and help get your PR merged quicker.
-
-When updates to your PR are requested, please add new commits and do not squash the history.
-This will make it easier to identify new changes.
-The PR will be squashed anyways when it is merged.
-Thanks.
-
-For fast feedback, please @-mention maintainers that are listed in the Chart.yaml file.
-
-Please make sure you test your changes before you push them.
-Once pushed, GitHub Actions will run across your changes and do some initial checks and linting.
-These checks run very quickly.
-Please check the results.
-We would like these checks to pass before we even continue reviewing your changes.
--->
 <!-- markdownlint-disable MD041 -->
 
-### What this PR does / why we need it
+### What does this PR do?
 
-### Which issue this PR fixes
+<!-- Describe the purpose of this PR, and any background context. -->
 
-*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
```

Comment:
Why remove this?

Reply:
Assumed people would know about the common keywords, restored 👀

Reply:
lots of people don't